const.go

```diff
@@ -218,6 +218,8 @@ const (
 	BlockTypeChildPage     BlockType = "child_page"
 	BlockTypeChildDatabase BlockType = "child_database"
 
+	BlockTypeAudio BlockType = "audio"
```

Comment:
Other constants are grouped. Maybe then put this `BlockTypeAudo` after `image` and before `video` so it's grouped as well?